vanilla-framework/vanilla-framework
# Add social icons to icon set

## Done

- Add social icons to icon set
- Added docs for all icons

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-social/
- Check icons and hover states
- Check docs are accurate

## Details

Fixes #1393
